modulemd/v2/tests/ModulemdTests/translation_helpers.py

```diff
@@ -68,6 +68,52 @@ def test_translation_catalog_from_index(self):
         # There are 34 unique summaries and descriptions in f29.yaml
         self.assertEquals(len(catalog), 34)
 
+    def test_translations_from_catalog(self):
+        translation_files = ["../test_data/nl.po"]
+
+        catalogs = dict()
```

Comment:
As discussed on IRC, don't make this a dictionary. Just a `list()` will be sufficient.